platforms/extensibility/unit-test-fixtures/src/test/resources/META-INF/gradle-plugins/org.gradle.custom-plugin.properties

```diff
@@ -0,0 +1 @@
+implementation-class=org.gradle.testfixtures.CustomPlugin
```

Comment:
Why add this now? And let's add a newline.

Reply:
This is a test resource that was moved from `core`. Not sure why the diff thinks it's a new file.

Reply:
Actually, I do know why. There were 2 independent unit tests that were sharing this resource and those unit tests moved into separate projects, so I copied the resource (rather than, say, refactoring the tests to not need a resource).